docs/migration.md

```diff
@@ -0,0 +1,150 @@
+
+## Switch dependent packages
+
+Open your app's package.json file and replace the ACP-prefixed packages with the corresponding AEP-prefixed packages. For example, the following code snippet shows the changes for `core` library:
+```diff
+...
+
+"dependencies": {
+    "react-native": "0.64.2",
+-   "@adobe/react-native-acpcore": "^2.0.0"
++   "@adobe/react-native-aepcore": "^1.0.0",
+    ...
+},
+
+```
+At this time, the following ACP-prefix libraries can be switched out with their respective AEP-prefix libraries.  However, ACP and AEP React Native libraries are not compatible. For extensions not supported in AEP-prefixed libraries, you should remove those packages from your package.json file. 
+
+| React Native (ACP 2.x) | React Native (AEP 1.x) |
+| :--- | :--- |
+| @adobe/react-native-acpcore | @adobe/react-native-aepcore |
+| @adobe/react-native-acpuserprofile | @adobe/react-native-aepuserprofile |
+| @adobe/react-native-acpplaces | NA |
+| @adobe/react-native-acpplaces-monitor | NA |
+| @adobe/react-native-acpanalytics | NA |
+| @adobe/react-native-acpmedia | NA |
+| @adobe/react-native-acpaudience | NA |
+| @adobe/react-native-acptarget | NA |
+| @adobe/react-native-acpcampaign | NA |
+| @adobe/react-native-aepassurance:2.x (compatible with ACP libraries) | @adobe/@adobe/react-native-aepassurance:3.x (compatible with AEP libraries)|
+
+<!--- TODO: add more descritions for Assurance library?? --->
+
+## Update SDK initialization
+Remove the registration code for extensions that are not supported in AEP React Native libraries.
+### Android
+```diff
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.Identity;
+import com.adobe.marketing.mobile.InvalidInitException;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Signal;
+import com.adobe.marketing.mobile.UserProfile;
+...
+import android.app.Application;
+...
+public class MainApplication extends Application implements ReactApplication {
+  ...
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setLogLevel(LoggingMode.DEBUG);
+    MobileCore.setWrapperType(WrapperType.REACT_NATIVE);
+
+    try {
+      UserProfile.registerExtension();
+      Identity.registerExtension();
+      Lifecycle.registerExtension();
+      Signal.registerExtension();
+-     Analytics.registerExtension();
+-     Target.registerExtension();
+-     Places.registerExtension();
+-     Campaign.registerExtension();
+      MobileCore.start(new AdobeCallback () {
+          @Override
+          public void call(Object o) {
+            MobileCore.configureWithAppID("<your_environment_id_from_Launch>");
```

Comment:
nit: Can we remove reference to launch
<your_environment_id_from_datacollectionUI>
<your_environment_id_of_mobile_property>
<your_environment_id_of_tag_property>

whatever that suits

Reply:
Did you change for this one?

Reply:
Ha, I only updated the iOS code, I will fix this as well.